.github/workflows/test.yml

```diff
@@ -80,6 +81,10 @@ jobs:
             domain: 'dnsimple-3-2-terraform.bio'
             registrant_contact_id: 14323
             registrant_change_domain: 'dnsimple-tf-ci-3.eu'
+          - terraform: '1.12.*'
+            domain: 'dnsimple-3-3-terraform.bio'
+            registrant_contact_id: 14323
+            registrant_change_domain: 'dnsimple-tf-ci-4.eu'
```

Comment:
If I recall correctly there is some setup that needs to be done for the domains we use in our integrations tests.

Reply:
There is no documentation in this repo, or in the internal wiki about maintaining these. Do you have any personal notes to add for this?

Reply:
I rotated this the deps instead of adding, and so could re-ruse but I would like advice on how I should document this, do you agree with my decision?